File: Codewars/python/Fizz Buzz.py
def solution(number):
    number-=1
    return [int(number/3*(4/5)),int(number/5*(2/3)),int(number/15)]

def fizzbuzz(n):
    return ["Fizz"[n%3*4:]+"Buzz"[n%5*4:] or n for n in range(1,n+1)]

File: Leetcode/cpp/Fizz Buzz.cpp
#include <string>
class Solution {
public:
    vector<string> fizzBuzz(int n) {
    vector<string> fizzbuzz;
    for (int i = 1; i < n+1; ++i) {
        if (i%3 == 0) {
            if (i%5 == 0) {
                fizzbuzz.push_back ("FizzBuzz");
            } else {
                fizzbuzz.push_back ("Fizz");
            }
        } else {
            if (i%5 == 0) {
                fizzbuzz.push_back ("Buzz");
            } else {
                fizzbuzz.push_back (to_string(i));
            }
        }
    }
    return fizzbuzz;
    }
};